plugins/@grouparoo/bigquery/src/lib/connect.ts

```diff
@@ -6,6 +6,9 @@ export const connect: ConnectPluginAppMethod = async ({ appOptions }) => {
   const projectId = appOptions.project_id?.toString() || "";
   const dataset = appOptions.dataset?.toString() || "";
   const client_email = appOptions.client_email?.toString() || "";
+  const datasetOptions = appOptions.location
+    ? { location: appOptions.location?.toString() }
+    : {};
```

Comment:
If the location is now required, this could become:

```suggestion
  const datasetOptions = { location: appOptions.location.toString() };
```

Reply:
Issuse